packages/storage-plus/README.md

```diff
@@ -113,7 +113,8 @@ iteration. That's right, you can list all items in a `Map`, or only
 part of them. We can efficiently allow pagination over these items as
 well, starting at the point the last query ended, with low gas costs.
 This requires the `iterator` feature to be enabled in `cw-storage-plus`
-(which automatically enables it in `cosmwasm-std` as well).
+(which automatically enables it in `cosmwasm-std` as well, and which is
+enabled by default).
```

Comment:
IMO this sentence sounds bit confusing.

> We can efficiently allow pagination (...). This requires the `iterator` feature to be enabled in `cw-storage-plus` (which automatically enables it in `cosmwasm-std` as well, and which is enabled by default).

Maybe change to something like this?

> This requires the `iterator` feature to be enabled in `cw-storage-plus` (it is enabled by default; it also automatically enables it in `cosmwasm-std`).